edflow/explore.py

```diff
@@ -226,7 +228,13 @@ def show_example(dset, idx, config):
 
 
 def _get_state(config):
-    Dataset = get_obj_from_str(config["dataset"])
+    if contains_key(config, "dataset"):        
+        Dataset = get_obj_from_str(config["dataset"])
+    elif contains_key(config, "datasets/train"):
+        module_name = retrieve(config, "datasets/train")
+        Dataset = get_obj_from_str(module_name)
+    else:
+        raise KeyError
```

Comment:
Do you think we should also add the possiblity to explore the validation dataset? Maybe with a selector in the sidebar?

Reply:
I was only fixing a bug that got introduced here. I am not willing to implement a new feature at this point.

I think it is not important enough to implement this feature at the moment because you can just hack around it.